spec/models/user_spec.rb

```diff
@@ -2,25 +2,25 @@
 
 RSpec.describe User, :type => :model do
   it "has role" do
-    expect(User.new(role: "Standard")).to have_attributes(role: "Standard")
+    expect(User.new(role: "standard")).to have_attributes(role: "standard")
   end
 
   it "has standard role by default" do
-    expect(User.new).to have_attributes(role: "Standard")
+    expect(User.new).to have_attributes(role: "standard")
   end
 
   it "has role set to standard" do
-    user = User.new(role: "Standard")
+    user = User.new(role: "standard")
     expect(user.standard?).to eq(true)
   end
 
   it "has role set to premium" do
-    user = User.new(role: "Premium")
+    user = User.new(role: "premium")
     expect(user.premium?).to eq(true)
   end
 
   it "has role set to admin" do
-    user = User.new(role: "Admin")
+    user = User.new(role: "admin")
     expect(user.admin?).to eq(true)
   end
```

Comment:
Add tests for the opposite situation, to prove that `.admin?` etc. can be false. Currently all of these methods would pass if the implementation was like this:

```ruby
def admin?
  true
end
def premium?
  true
end
def standard?
  true
end
```

An example to add missing test coverage for `admin?`. This would replace lines 22-25 in the spec:

```ruby
context "admin?" do
  it "returns false for non-admins" do
    non_admin = User.new(role: "premium")
    expect(user.admin?).to eq(false)
  end

  it "returns true for admins" do
    admin_user = User.new(role: "admin")
    expect(admin_user.admin?).to eq(true)
  end
end
```

This pattern can be followed for `premium?` and `standard?` to improve their test coverage.